install/deb/templates/web/nginx/php-fpm/phpbb.stpl

```diff
@@ -0,0 +1,117 @@
+# Default Web Domain Template                                             #
+# DO NOT MODIFY THIS FILE! CHANGES WILL BE LOST WHEN REBUILDING DOMAINS   #
+# https://hestiacp.com/docs/server-administration/web-templates.html      #
+#=========================================================================#
+
+server {
+	listen      %ip%:%web_ssl_port% ssl;
+	server_name %domain_idn% %alias_idn%;
+	root        %sdocroot%;
+	index       index.php index.html index.htm;
+	access_log  /var/log/nginx/domains/%domain%.log combined;
+	access_log  /var/log/nginx/domains/%domain%.bytes bytes;
+	error_log   /var/log/nginx/domains/%domain%.error.log error;
+
+	ssl_certificate     %ssl_pem%;
+	ssl_certificate_key %ssl_key%;
+	ssl_stapling        on;
+	ssl_stapling_verify on;
+
+	# TLS 1.3 0-RTT anti-replay
+	if ($anti_replay = 307) { return 307 https://$host$request_uri; }
+	if ($anti_replay = 425) { return 425; }
+
+	include %home%/%user%/conf/web/%domain%/nginx.hsts.conf*;
+
+	location = /favicon.ico {
+		log_not_found off;
+		access_log off;
+	}
+
+	location = /robots.txt {
+		allow all;
+		log_not_found off;
+		access_log off;
+	}
+
+	location ~ /\.(?!well-known\/) {
+		deny all;
+		return 404;
+	}
+
+	# Based on: https://github.com/phpbb/phpbb/blob/master/phpBB/docs/nginx.sample.conf
+	location / {
+		try_files $uri $uri/ @rewriteapp;
+
+		location ~* ^.+\.(ogg|ogv|svg|svgz|swf|eot|otf|woff|woff2|mov|mp3|mp4|webm|flv|ttf|rss|atom|jpg|jpeg|gif|png|webp|ico|bmp|mid|midi|wav|rtf|css|js|jar)$ {
+			fastcgi_hide_header "Set-Cookie";
+		}
+
+		# Pass the php scripts to FastCGI server specified in upstream declaration.
+		location ~ \.php(/|$) {
+			include         /etc/nginx/fastcgi_params;
+
+			fastcgi_index index.php;
+			fastcgi_split_path_info ^(.+\.php)(/.*)$;
+			fastcgi_param PATH_INFO $fastcgi_path_info;
+			fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
+			#fastcgi_param HTTP_EARLY_DATA $rfc_early_data if_not_empty;
+
+			fastcgi_pass %backend_lsnr%;
+			include %home%/%user%/conf/web/%domain%/nginx.fastcgi_cache.conf*;
+
+			try_files $uri $uri/ /app.php$is_args$args;
+		}
+
+		# Deny access to internal phpbb files.
+		location ~ /(config\.php|common\.php|cache|files|images/avatars/upload|includes|(?<!ext/)phpbb(?!\w+)|store|vendor) {
+			deny all;
+			return  404;
+		}
+	}
+
+	location @rewriteapp {
+		rewrite ^(.*)$ /app.php/$1 last;
+	}
+
+	# Correctly pass scripts for installer
+	location /install/ {
+		try_files $uri $uri/ @rewrite_installapp =404;
+
+		# Pass the php scripts to fastcgi server specified in upstream declaration.
+		location ~ \.php(/|$) {
+			include         /etc/nginx/fastcgi_params;
+
+			fastcgi_index index.php;
+			fastcgi_split_path_info ^(.+\.php)(/.*)$;
+			fastcgi_param PATH_INFO $fastcgi_path_info;
+			fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
+			#fastcgi_param HTTP_EARLY_DATA $rfc_early_data if_not_empty;
```

Comment:
Also here.

Reply:
It's true, when I polished the content of my test server I forgot to delete them.

Reply:
Update when you have time, thanks!